rasa/core/domain.py

```diff
@@ -672,40 +676,37 @@ def get_parsing_states(self, tracker: "DialogueStateTracker") -> Dict[Text, floa
         # be ignored for the current intent
         latest_message = tracker.latest_message
 
-        if not latest_message:
+        if not latest_message.text and not latest_message.intent_name:
```

Comment:
I think it should still be 
```suggestion
        if not latest_message:
```
it is checking for `None`